{"uuid":"52bd01d0-1246-4e09-87e3-17a305890c51","name":"Scenario Outline: The user getting error message with invalid python code for Types of Linked List","children":["52a7098a-a09b-42c8-9cc3-1beefd2889fb"],"befores":[],"afters":[{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696462843192,"stop":1696462843193},{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696462843311,"stop":1696462843311},{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696462843364,"stop":1696462843365},{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696462845388,"stop":1696462845388}],"start":1696462842983,"stop":1696462845389}